unix/w0vncserver/Pipewire.cxx

```diff
@@ -0,0 +1,411 @@
+#include "Pipewire.h"
+#include "pipewire/loop.h"
+#include "pipewire/properties.h"
+#include "pipewire/stream.h"
+#include "spa/param/video/raw.h"
+#include "spa/pod/builder.h"
+#include "spa/pod/pod.h"
+#include "spa/utils/type.h"
+#include <spa/param/latency-utils.h>
+#include <spa/debug/format.h>
+#include <spa/debug/pod.h>
+
+#include <pipewire/pipewire.h>
+#include <stdexcept>
+#include <assert.h>
+#include <core/LogWriter.h>
+
+#include "WDesktop.h"
+
+#include <pixman.h>
```

Comment:
We try to sort the includes in most files for readability:
1. Standard C headers
2. Standard C++ headers
3. External libraries
4. Internal libraries
5. Local headers